jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/web/WebScreenshotCreator.java

```diff
@@ -61,16 +61,29 @@ public int getHierarchy() {
 
     public void createScreenshot(Type screenShotType) {
         if (desiredMode.getHierarchy() >= screenShotType.getHierarchy()) {
-            try {
-                if (ExpectedConditions.alertIsPresent().apply(driver) != null) {
-                    log.info("Can't take screenshot (alert is present)");
-                } else if (driver instanceof TakesScreenshot) {
```

Comment:
Why did you removed check `if (driver instanceof TakesScreenshot)` ?

Reply:
We were on a simplifying spree and made a mistake :-D